ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewBackgroundDrawable.java

```diff
@@ -123,6 +126,20 @@ public int getOpacity() {
     return ColorUtil.getOpacityFromColor(ColorUtil.multiplyColorAlpha(mColor, mAlpha));
   }
 
+  @Override
+  public void getOutline(@NonNull Outline outline) {
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+      float borderRadius = (!CSSConstants.isUndefined(mBorderRadius) && mBorderRadius > 0)
+              ? mBorderRadius
+              : 0;
+      float extraRadiusFromBorderWidth = (mBorderWidth != null)
+              ? mBorderWidth.get(Spacing.ALL)/2
+              : 0;
+
+      outline.setRoundRect(getBounds(), borderRadius+extraRadiusFromBorderWidth);
+    }
```

Comment:
Can you please add a comment explaining why this is needed? Why is it needed? Isn't setting `elevation` enough?